.github/pull_request_template.md

```
@@ -45,6 +45,18 @@ Fixes #<!--Add related issue number here.-->
 
 <!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
 
+**Checklist**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)